Files/BaseLayout.cs

```diff
@@ -438,7 +438,6 @@ private async void FolderSettings_GroupOptionPreferenceUpdated(object sender, Ev
         protected override void OnNavigatingFrom(NavigatingCancelEventArgs e)
         {
             base.OnNavigatingFrom(e);
-            UnhookBaseEvents();
```

Comment:
Removed otherwise the method is not called anymore after navigating to another folder.
Alternative is to add call to `HookBaseEvents` in `OnNavigatedTo`, but `HookEvent` is handled differently.